projects/optic/src/commands/diff/diff.ts

```diff
@@ -208,13 +207,14 @@ const runDiff = async (
     }
 
     logger.info('');
-
-    if (!hasOpticUrl) {
+    if ((!hasOpticUrl && headFile.from === 'file') || headFile.from === 'git') {
```

Comment:
This was incorrect before:
- It logged when it was an empty spec
- It added a path that wasn't copy paste-able (i.e. it only showed the base path)
- It logged out the base file, not the head file path (i.e. could have been a `git:...`